pydicom/sopclass_uids.py

```diff
@@ -0,0 +1,299 @@
+"""
+Storage SOP Class UIDs auto-generated by generate_storage_sopclass_uids.py
+"""
```

Comment:
yes I think so! I was just going off of what the file is currently called. Specificity is always better of course... I don't think the files have any hangups about being a bit longer :)